src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/NoOnlineCancelAlert.jsx

```diff
@@ -23,21 +27,34 @@ export default function NoOnlineCancelAlert({ appointment, facility }) {
       backgroundOnly
     >
       {!facility &&
+        !isCompAndPenAppointment &&
         'To reschedule or cancel this appointment, contact the VA facility where you scheduled it.'}
       {!!facility &&
+        !isCompAndPenAppointment &&
```

Comment:
This always throws me for a loop. What's the difference between the 1st !facility and the 2nd !!facility?

Reply:
!facility will return true if facility data is undefined or null. 
!!facility will return false